.github/workflows/ci.yml

```diff
@@ -110,6 +110,30 @@ jobs:
         with:
           job-name: Base IT
 
+  quarkus-unit-tests:
+    name: Quarkus UT
+    needs: build
+    timeout-minutes: 15
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+
+      # We also want to download Keycloak artifacts
+      - id: integration-test-setup
+        name: Integration test setup
+        uses: ./.github/actions/integration-test-setup
```

Comment:
We don't want to rebuild all dependent modules specified by the `-am` flag. We want to use already-built artifacts from the `build` phase. I'll make improvements to it in different PR. ATM, we can have the Quarkus UT working.